data-service/templates/data_service.yml

```diff
@@ -2,60 +2,68 @@ AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
 Resources:
 
-  EnvironmentsTable:
+  EnvironmentTable:
     Type: AWS::DynamoDB::Table
     Properties:
      #TODO: change ddb table names to have region prefix passed in as a resource
      TableName: Environments
      AttributeDefinitions:
        - AttributeName: environmentId
          AttributeType: S
-       - AttributeName: environmentVersion
+       - AttributeName: environmentName
+         AttributeType: S
+       - AttributeName: accountId
+         AttributeType: S
+       - AttributeName: cluster
          AttributeType: S
      KeySchema:
        - AttributeName: "environmentId"
```

Comment:
This won't be an `environmentId` anymore right? The hash key should be `accountId + clusterName` (no environment-specific value there) which unambiguously identifies a cluster, so perhaps `clusterId`?

Reply:
yea i was going to name the field environmentId in the ddb table but its going to be constructed out of accountId_clusterName